src/main/java/pl/durilian/wordTermsChecker/pages/NewPage.java

```diff
@@ -8,7 +8,7 @@
 
 @FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
 public class NewPage extends AbstractBasePage<NewPage> {
-    public final static String URI = "/new";
+    public static String URI = "/new";
```

Comment:
Co daje to pole? Takie samo (ale dla instancji a nie statyczne) jest w klasie nadrzędnej.